src/survey.ts

```diff
@@ -7390,6 +7402,7 @@ export class SurveyModel extends SurveyElementCore
           }
         }
       } else if (isStrCiEqual(layoutElement.id, "advanced-header")) {
+        const header = layoutElement.data as Cover;
```

Comment:
const header is not used in this scope. maybe we can delete this line?

Reply:
Forgot it. Removed